code/modules/mob/living/carbon/human/species_types/eggpeople.dm

```diff
@@ -0,0 +1,42 @@
+// This is the dumbest fucking thing I have ever programmed t. Altoids
+#define EGG_MAXBRUTEMOD 1.5
+#define EGG_ALPHA 0.00018
+#define EGG_BETA (-0.027)
+/datum/species/egg
+	// Egghead humanoids composed of eggs and stale memes
+	name = "Eggperson"
+	id = "egg"
+	say_mod = "blurbles"
+	fixed_mut_color = "FFE7C9"
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,0), OFFSET_EARS = list(0,2), OFFSET_SHOES = list(0,0), OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,0), OFFSET_HEAD = list(0,2), OFFSET_FACE = list(0,0), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,0))
```

Comment:
No need for the offsets that are list(0,0) , mobcode doesnt apply the offset if it's not in the list

Reply:
Oh that's nifty!